django/db/models/functions/text.py

```diff
@@ -235,6 +236,54 @@ class RTrim(Transform):
     lookup_name = 'rtrim'
 
 
+class SHA1(MD5):
+    function = 'SHA1'
+    lookup_name = 'sha1'
+
+    def as_postgresql(self, compiler, connection, **extra_content):
+        # digest() function requires the pgcrypto extension.
```

Comment:
It's unclear to me why this comment is here.

Reply:
As you feel it is not required, consider it is gone.